docs/cave/d2d-gis-shapefiles.md

```diff
@@ -5,10 +5,12 @@ This section describes how to start the GIS application, configure the display a
 
 ## Display GIS Data
```

Comment:
The very first section of this page ends with "The following topics are discussed:" and then doesn't actually provide a list of the sections.  I would add a bulleted list of the sections and add their corresponding subheading links as well.

Reply:
I removed the whole two sentences and then added the bullets since it was repetitive